system/jhtree/jhtree.hpp

```diff
@@ -38,22 +38,49 @@ interface jhtree_decl IDelayedFile : public IInterface
     virtual IFileIO *getFileIO() = 0;
 };
 
+class KeyStatsCollector
+{
+public:
+    IContextLogger *ctx;
+    unsigned seeks = 0;
+    unsigned scans = 0;
+    unsigned wildseeks = 0;
+    unsigned skips = 0;
+    unsigned nullskips = 0;
```

Comment:
trivial: could delete since no longer returned.

Reply:
It's returned via the log context still